### DISCUSSION TITLE
How to update cockpit-machines package?

### DISCUSSION THREAD

**Discussion by chall3ng3r** [cockpit-project/cockpit-machines#540] [Q&A] [answered]
Hello,

I installed cockpit and machines package through `apt install cockpit-machines` command on Linux Mint 20.3. It installed v215 instead of the latest one, or even a ver behind latest. 

Cockpit main also updated to v261, but I have v260-1~bpo20.04.1.

How I can install updates to latest version?

**Comment by chall3ng3r** [comment, marked as answer]
never mind, I looked up install instructions again and noticed backports, tried same with machines package, `sudo apt install -t ${UBUNTU_CODENAME}-backports cockpit-machines` and it updated it to v259.

VMs page was crashing before this, now it started working as expected.

**Comment by smoke007** [comment]
For anyone else looking for how to get a newer version installed on Linux Mint; here is the command:

```
. /etc/os-release
sudo apt install -t ${UBUNTU_CODENAME}-backports cockpit cockpit-machines 
```
I requested for this to be added to the install directions page:
[https://github.com/cockpit-project/cockpit-project.github.io/pull/795](url)
